tests/conftest.py

```diff
@@ -176,7 +176,7 @@ def check_import() -> bool:
 
 
 @pytest.fixture(autouse=True)
-def set_event_loop() -> Iterator[None]:
+def set_event_loop(anyio_backend) -> Iterator[None]:  # type: ignore[reportUnknownParameterType]
```

Comment:
I think anyio_backend is either a str or tuple?

```suggestion
def set_event_loop(anyio_backend: str) -> Iterator[None]:
```